### PR TITLE
Completed home page

### PR DESCRIPTION
**Pull request by aayushjain-code**

### CODE REVIEW

**Comment by aayushjain-code** [comment]
live: https://focused-brattain-951859.netlify.app/

File: css/Category.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+	  position: absolute;
+	  top: 50%;
+	  left: 40%;
+	  transform: translate(-50%, -50%);
+}
+
+.Category .box-container .box .content h3 {
+	background-color: rgba(236, 236, 236, 0.705);
+	padding: 0.5rem 0.5rem;
```
Comment (ApurvaSawant11) [inline css/Category.css]: You can just write `padding : 0.5rem`

File: css/buttons.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+.btn-secondary {
+	margin-top: 1rem;
+	display: inline-block;
+	padding: .8rem 2rem;
+	font-size: 1.4rem;
+	color: #fff;
+	background: var(--secondary);
+	border-radius: .5rem;
+	cursor: pointer;
```
Comment (ApurvaSawant11) [inline css/buttons.css]: properties are same for `.btn` and `.btn-secondary`. You can use a comma separator for the same properties (like .btn, .btn-secondary { ..... }) and separately define `background` property.

File: css/Category.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+.Category .box-container .box {
+	background: var(--bg-color-1);
+	border-radius: .5rem;
+	box-shadow: var(--box-shadow);
+	padding: 2rem;
+	position: relative;
+}
+
+.Category .box-container .box .image {
```
Comment (partha8) [inline css/Category.css]: don't use capitalized classes in html/css.

File: css/general.css [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+	text-align: center;
+	padding-bottom: 2rem;
+	color: var(--text-color-1);
+	font-size: 4rem;
+}
+
+.heading span {
+	position: relative;
+	z-index: 0;
+}s
```
Comment (ApurvaSawant11) [inline css/general.css]: I guess this is by mistake

**Review by ApurvaSawant11 — APPROVED** [review]
UI looks good. Nice hover effect on featured categories.

Just make a few suggested changes

File: css/header.css [hunk 5]
```diff
@@ -0,0 +1,163 @@
+.header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	right: 0;
+	background: var(--bg-color-1);
+	box-shadow: var(--box-shadow);
+	padding: 1.5rem 7%;
+	z-index: 1000;
+	display: flex;
```
Comment (anik31) [inline css/header.css:10]: such high values of z-index isn't required

**Review by partha8 — APPROVED** [review]
I have also noticed that you are being very specific when you style the elements. While I understand why you did it after going through the code, you should try to have different classes for different components and reuse classes whenever possible. 

I am approving here but make the change that i suggested in the previous comment.

File: css/Category.css (hunk 3 above)
Comment (aayushjain-code) [reply css/Category.css]: Thank you for bringing my attention I will fix it for sure!

File: css/buttons.css (hunk 2 above)
Comment (aayushjain-code) [reply css/buttons.css]: Correct i will implement this using comma as you said✅

File: css/general.css (hunk 4 above)
Comment (aayushjain-code) [reply css/general.css]: yes!  i was trying to implement some styles but forgot to remove them.

File: css/header.css (hunk 5 above)
Comment (aayushjain-code) [reply css/header.css:10]: Got it thanks for the suggestion!

File: css/Category.css (hunk 1 above)
Comment (aayushjain-code) [reply css/Category.css]: Got it Apurva will do it!

**Comment by aayushjain-code** [comment]
done with the  changes and pr reviews suggestions thanks @anik31  @ApurvaSawant11  @partha8  for reviews.

**Pull request merged**